CRM/Core/IDS.php

```diff
@@ -110,7 +110,7 @@ public static function createBaseConfig() {
         'filter_type' => 'xml',
         'filter_path' => "{$pkgs}/IDS/default_filter.xml",
         'tmp_path' => $tmpDir,
-        'HTML_Purifier_Path' => $pkgs . '/IDS/vendors/htmlpurifier/HTMLPurifier.auto.php',
+        'HTML_Purifier_Path' => '[civicrm.root]/vendor/ezyang/htmlpurifier/library/HTMLPurifier.auto.php',
```

Comment:
Ditto, couldn't we just leave this blank and let the autoloader do its thing?

Reply:
@colemanw no I don't believe so because the IDS doesn't really understand things in composer land

Also this needs to use the ['civicrm.vendor'] path and also that will just print literal ['civicrm.root'] not the calculated path I believe.

Reply:
OK cool. The `['civicrm.vendor']` must be new since the last time this was attempted. I was just copying the code from there.

I don't think it will print the literal string because of this: 

https://github.com/civicrm/civicrm-packages/blob/7db3890327e396abc04138fdd54d5c05a827ec70/IDS/Monitor.php#L207-L213